flink-staging/flink-table/pom.xml

```diff
@@ -37,7 +37,7 @@ under the License.
 		<dependency>
 			<groupId>com.google.guava</groupId>
 			<artifactId>guava</artifactId>
-			<version>${guava.version}</version>
+			<version>14.0.1</version>
```

Comment:
What exactly was the problem with the hcat input format and guava?
If their guava version is really incompatible with ours, we can do the following:
- Use a newer hcat version
- create a special shaded-hcat maven module which shades away hcat's guava.